changelog/8569.removal.md

```diff
@@ -0,0 +1 @@
+Removed Python 3.6 support
```

Comment:
I think we could mention that we're removing 3.6 support because it's reaching its end of life by the end of this year:
```suggestion
Removed Python 3.6 support as [it reaches its end of life in December 2021](https://www.python.org/dev/peps/pep-0494/#lifespan).
```
WDYT?